src/main/app/claims/claimStoreClient.ts

```diff
@@ -69,7 +69,11 @@ export default class ClaimStoreClient {
       .get(`${claimStoreApiUrl}/${externalId}`)
       .then(claim => {
         if (claim) {
-          return new Claim().deserialize(claim)
+          if (userId !== claim.submitterId) {
+            throw new Error('You are not allowed to access this resource')
```

Comment:
Would be nice if this was e.g. a `ForbiddenError` which would map to a forbidden page.